### ISSUE TITLE
Failed to install due to `postinstall` script on v0.7.3

### ISSUE THREAD

**Issue by mmyoji** [scottrippey/next-router-mock#56]
Tring to install/upgrade to v0.7.3, `npm install` fails due to `postinstall` script added on 16dd116a53abee422b647022087f202c43b36708.

Does this necessary for users of this package?

```
$ npm i -D next-router-mock
npm ERR! code 2
npm ERR! path /home/mmyoji/tmp/node_modules/next-router-mock
npm ERR! command failed
npm ERR! command sh -c (cd test/next-10 && npm i); (cd test/next-11 && npm i); (cd test/next-12 && npm i); (cd test/next-canary && npm i)
npm ERR! sh: 1: cd: can't cd to test/next-10
npm ERR! sh: 1: cd: can't cd to test/next-11
npm ERR! sh: 1: cd: can't cd to test/next-12
npm ERR! sh: 1: cd: can't cd to test/next-canary

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/mmyoji/.npm/_logs/2022-06-13T02_20_43_937Z-debug-0.log
```

**Comment by scottrippey** [comment]
Sorry about that, I forgot `postinstall` runs for consumers too!  
Fixed in `0.7.4`

**Comment by mmyoji** [comment]
Thank you for the quick fix! 😄 ✨
